keyboards/handwired/2d4d/info.json

```diff
@@ -0,0 +1,85 @@
+{
+    "keyboard_name": "2d4d",
+    "maintainer": "qmk",
+    "manufacturer": "mechMehmet",
+    "development_board": "blackpill_f401",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "features": {
+        "encoder": true,
```

Comment:
If encoders are enabled at the keyboard level, there needs to be some keyboard level encoder functionality as well. See [doc](https://docs.qmk.fm/#/feature_encoders?id=callbacks).